### ISSUE TITLE
Add an overlay that obscures Mesons a la AI no camera vision

### ISSUE THREAD

**Issue by Probe1** [d3athrow/vgstation13#13007]

**Comment by Shadowmech88** [comment]
I'm not certain that this is possible with the way mesons work.

**Comment by Intigracy** [comment]
obscures mesons in what context

**Comment by ghost** [comment]
This shouldn't be done by mapping

**Comment by Probe1** [comment]
I mean it should be done via coding and used for mapping.

It's a mapping issue.  The issue regards mapping.

**Comment by Exxion** [comment]
What do you mean?

**Comment by Probe1** [comment]
Players can see hidden content because turfs are visible on mesons.  That makes mapping hidden rooms pretty dodgy.

**Comment by Exxion** [comment]
But then seeing the overlay would tell you that there's hidden stuff there.

**Comment by Probe1** [comment]
Yes but it wouldn't tell you the exact nature and layout of what is ahead.

I know it's not perfect but it's _something_

**Comment by Exxion** [comment]
Alright, I guess that's probably doable
Though it could still be fucked with CheatEngine
